test/amqp-integration-test.py

```diff
@@ -46,12 +46,26 @@ def run(path):
     processes.append(info)
     return info
 
+def runDNS():
```

Comment:
It looks like the only way this function differs from the existing run is the removal of `-tags` and the `--config` path. I think the tags can be applied harmlessly to this build, and the `--config` arg can be factored out into a second argument. Could you consolidate the two functions?